client/util_1.11.go

```diff
@@ -0,0 +1,13 @@
+// +build go1.11
```

Comment:
Am I missing something or is that now identical to the 1.10 one?

Reply:
Oops, that was meant for testing only. The 1.10 version should import the other vsock package. I'll fix that.